src/python/WMCore/WorkQueue/WorkQueueBackend.py

```diff
@@ -416,7 +416,7 @@ def availableWork(self, thresholds, siteJobCounts, teams=None, wfs=None,
                 siteJobCounts[possibleSite][prio] = siteJobCounts[possibleSite].setdefault(prio, 0) + \
                                                     element['Jobs'] * element.get('blowupFactor', 1.0)
             else:
-                self.logger.info("No available resources for %s with doc id %s", element['RequestName'], element.id)
+                self.logger.debug("No available resources for %s with doc id %s", element['RequestName'], element.id)
```

Comment:
Alan, shouldn't this be separate patch and need to be applied not for the testing?

Reply:
agreed! I'm making a new patch to apply with this one